howfairis/repo.py

```diff
@@ -13,8 +14,20 @@ class Repo:
             Can also be a commit SHA-1 hash or tag.
         path: Path inside repository. Defaults to root.
 
+    Attributes:
+        url (str): URL of a code repository,
+        branch (str, None): Branch to checkout. If None then :attr:`Repo.default_branch` will be used.
+        path (str): Path inside repository.
```

Comment:
```suggestion
        path (str, None): Path inside repository.
```

Reply:
Nope it can be None see 
https://github.com/fair-software/howfairis/blob/11b17096803f655ae23a6bd2b78d3005795294f2/tests/github/repo1/test_repo_no_args.py#L20

Reply:
indeed, `branch` can be `None`, but I was referring to `path` (which can also be `None`). Perhaps I'm missing something?

Reply:
Sorry, path is always a string see
https://github.com/fair-software/howfairis/blob/60396e58a74cb53b28856f60103d84c9631db521/howfairis/repo.py#L24

Reply:
OK I see what you mean, but I think the docs should describe the constructor's signature, right? And not take into account anything that comes after.

Reply:
`Args:` section describes constructor signature and the `Attributes:` section the properties of an object of that class.

Reply:
OK, so after you instantiate `Repo`, `path` is supposed to always be a `str`

Reply:
yep